src/main/java/com/google/sps/api/GmailConfiguration.java

```diff
@@ -0,0 +1,128 @@
+
+  /**
+    * Create a MimeMessage using the parameters provided.
+    *
+    * @param to email address of the receiver
+    * @param from email address of the sender, the mailbox account
+    * @param subject subject of the email
+    * @param bodyText body text of the email
+    * @return the MimeMessage to be used to send email
+    * @throws MessagingException
+    */
+  public static MimeMessage createEmail(String to,
+                                        String from,
+                                        String subject,
+                                        String bodyText)
+      throws MessagingException {
+    Properties props = new Properties();
+    Session session = Session.getDefaultInstance(props, null);
+
+    MimeMessage email = new MimeMessage(session);
+
+    email.setFrom(new InternetAddress(from));
+    email.addRecipient(javax.mail.Message.RecipientType.TO,
+            new InternetAddress(to));
+    email.setSubject(subject);
+    email.setText(bodyText);
+    return email;
+  }
+
+  /**
+    * Create a message from an email.
+    *
+    * @param emailContent email to be set to raw of message
+    * @return message containing a base64url encoded email
+    * @throws IOException
+    * @throws MessagingException
+    */
+  public static Message createMessageWithEmail(MimeMessage emailContent)
+      throws MessagingException, IOException {
+    ByteArrayOutputStream buffer = new ByteArrayOutputStream();
+    emailContent.writeTo(buffer);
+    byte[] bytes = buffer.toByteArray();
+    String encodedEmail = Base64.encodeBase64URLSafeString(bytes);
+    Message message = new Message();
+    message.setRaw(encodedEmail);
+    return message;
+  }
+
+  /**
+    * Send an email from the user's mailbox to its recipient.
+    *
+    * @param service authorized Gmail API instance.
+    * @param userId user's email address, using the special value "me"
+    * can be used to indicate the authenticated user.
+    * @param emailContent email to be sent.
+    * @return sent message
+    * @throws MessagingException
+    * @throws IOException
+    */
+  public static Message sendMessage(Gmail service,
+                                    String userId,
+                                    MimeMessage emailContent)
+      throws MessagingException, IOException {
+    Message message = createMessageWithEmail(emailContent);
+    message = service.users().messages().send(userId, message).execute();
+    return message;
+  }
+
+  /**
+    * Send a test email from areeta@google.com.
+    *
+    * @throws IOException
+    * @throws GeneralSecurityException
+    * @throws MessagingException
+    */
+  public static void sendEmail() 
+      throws IOException, GeneralSecurityException, MessagingException {
+		Gmail service = GmailAPI.getGmailService();
+		MimeMessage Mimemessage = createEmail("areeta@google.com","me","This my demo test subject","This is my body text");
```

Comment:
In the future you could make variables (ex String email = areeta@google.com, etc..) and then put those variables in the createEmail function instead of the strings themselves.

Reply:
Over here, I included as everything as strings just to test it and plan to use variables in the future!